src/Equations/AnalyticDistribution.cpp

```diff
@@ -0,0 +1,120 @@
+/**
+ * Functions to evaluate the analytic pitch-angle distribution. 
+ * (maybe other analytic distributions will be added later?)
+ */
+
+#include "DREAM/Equations/AnalyticDistribution.hpp"
+#include "FVM/config.h"
+#include "DREAM/Equations/EffectiveCriticalField.hpp"
+#include "DREAM/Equations/RunawayFluid.hpp"
+
+using namespace DREAM;
+
+/**
+ * Constructor.
+ *
+ */
+AnalyticDistribution::AnalyticDistribution(FVM::RadialGrid *rGrid, PitchScatterFrequency *nuD, OptionConstants::collqty_Eceff_mode Eceff_mode) 
+: rGrid(rGrid), nuD(nuD), Eceff_mode(Eceff_mode){}
+
+// ok, maybe re-name the constant so it doen't belong to Eceff?
```

Comment:
You could add a proper comment for the method here. Here's a template:
```
/**
 * DESCRIPTION
 *
 * ir:         DESC
 * xi0:        DESC
 * p:          DESC
 * Eterm:      DESC
 * inSettings: DESC
 * gsl_ad_w:   DESC
 */
```

Reply:
@Embreus wrote this function, so I would suggest he takes over this fix.